enterprise/backend/test/metabase_enterprise/serialization/v2/e2e_test.clj

```diff
@@ -735,3 +735,56 @@
                                           (cmd/v2-load! dump-dir {}))
                         "throws")))))))))))
+
+(deftest pivot-export-test
+  (testing "Pivot table export and load correctly"
+    (let [card1s (atom nil)]
+      (ts/with-random-dump-dir [dump-dir "serdesv2-"]
+        (ts/with-source-and-dest-dbs
+          (ts/with-source-db
+            (mt/dataset sample-dataset
+              (mt/$ids nil
+                (t2.with-temp/with-temp
+                  [Database   {db-id :id}    {:name "Pivot Database"}
+                   Collection {coll-id :id}  {:name "Pivot Collection"}
+                   Card       card           {:name          "Pivot Card"
+                                              :collection_id coll-id
+                                              :dataset_query {:type     :query
+                                                              :database db-id
+                                                              :query    {:source-table $$orders
+                                                                         :aggregation  [:sum [:field %orders.id nil]]
+                                                                         :breakout     [[:field %orders.user_id nil]]}}
+                                              :visualization_settings
+                                              {:pivot_table.column_split
+                                               {:rows    [[:field %people.name {:base-type    :type/Text
+                                                                                :source-field %orders.user_id}]]
+                                                :columns [[:field %products.title {:base-type    :type/Text
+                                                                                   :source-field %orders.product_id}]]
+                                                :values  [[:aggregation 0]]}
+                                               :column_settings
+                                               {(format "[\"ref\",[\"field\",%s,null]]" %people.name)
+                                                {:pivot_table.column_sort_order "descending"}}}}]
+                  (reset! card1s card)
+                  (-> (serdes/with-cache (into [] (extract/extract {})))
+                      (storage/store! dump-dir))))))
+
+          (ts/with-dest-db
+            (mt/dataset sample-dataset
+              (mt/$ids nil
+                (serdes/with-cache (serdes.load/load-metabase! (ingest/ingest-yaml dump-dir)))
+
+                (let [viz (t2/select-one-fn :visualization_settings Card :entity_id (:entity_id @card1s))]
+                  (is (contains? viz :pivot_table.column_split))
+                  ;; column ids are different in different dbs
+                  (is (not= (get-in @card1s [:visualization_settings :pivot_table.column_split :rows 0 1])
+                            (get-in viz [:pivot_table.column_split :rows 0 1])))
```

Comment:
better to use `testing` blocks to add meaningful error descriptions instead of just leaving a comment
```suggestion
                  (testing "check column ids are different in different dbs, because otherwise this test could be a false negative"
                    (is (not= (get-in @card1s [:visualization_settings :pivot_table.column_split :rows 0 1])
                              (get-in viz [:pivot_table.column_split :rows 0 1]))))
```